app/Http/Controllers/PostsController.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Post;
+use DB;
+
+class PostsController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+
+    // index => listing of all the posts
```

Comment:
These comments are redundant. What's the problem with redundancy? The cost of keeping them up to date, without any benefits.

Reply:
Ok. I figure that out